### DISCUSSION TITLE
Hello World - 没有秃头基因的燊的博客

### DISCUSSION THREAD

**Discussion by giscus** [yzs020220/giscus-repo#1] [Announcements]
# Hello World - 没有秃头基因的燊的博客

Welcome to Hexo! This is your very first post. Check documentation for more info. If you get any problems when using Hexo, you can find the answer in troubleshooting or you can ask me on GitHub.  Quic

https://yzs020220.github.io/2022/12/27/hello-world/

**Comment by yzs020220** [comment]
test:smile:
